betterproto/tests/generate.py

```diff
@@ -11,27 +13,38 @@
 os.environ["PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION"] = "python"
 
 
+def clear_directory(path: str):
+    for file_or_directory in glob.glob(os.path.join(path, '*')):
```

Comment:
Not too big a deal, but using pathlib would make some of this code a bit neater.
e.g.
```python
def clear_directory(path: Path):
    for file_or_directory in path.glob("*"):
        if file_or_directory.is_dir():
            shutil.rmtree(file_or_directory)
        else:
            file_or_directory.unlink()
```

Reply:
i see.. ok. i will change this as part of one of the other PR's that will be upcoming, and use pathlib from now on. hope this can be non-blocking for now